PINCache/PINOperationQueue.m

```diff
@@ -98,6 +105,7 @@ - (instancetype)initWithMaxConcurrentOperations:(NSUInteger)maxConcurrentOperati
     _highPriorityOperations = [[NSMutableOrderedSet alloc] init];
     
     _referenceToOperations = [NSMapTable weakToWeakObjectsMapTable];
+    _identifierToOperations = [NSMapTable weakToWeakObjectsMapTable];
```

Comment:
I think we're good here because the operation already keeps a strong reference to its identifier.